### PR TITLE
[PFX-772] Add JSDoc Typechecking to create-gasket-app

### PR DESCRIPTION
**Pull request by jpina1-godaddy**
## Summary

Add JSDoc Typechecking to create-gasket-app

## Changelog

create-gasket-app
- Added new types and type file
- Added typechecking 

## Test Plan

Added typecheck

### CODE REVIEW

File: packages/create-gasket-app/lib/index.d.ts [hunk 1]
```diff
@@ -25,6 +26,43 @@ export interface PackageJson extends Dependencies {
   homepage?: string;
 }
 
+export function commasToArray(value: string): string[];
```
Comment (agerard-godaddy) [inline packages/create-gasket-app/lib/index.d.ts:29]: Should this be in the internal types?

**Review by agerard-godaddy — APPROVED** [review]
small question/tweak - otherwise 👍

File: packages/create-gasket-app/lib/index.d.ts (hunk 1 above)
Comment (jpina1-godaddy) [reply packages/create-gasket-app/lib/index.d.ts:29]: Yeah I can move it to internal 👍

**Review by kbader-godaddy — APPROVED** [review]
Great work! Woot!

**Pull request merged**